src/schedule/zephyr_ll.c

```diff
@@ -15,8 +15,10 @@
 
 #include <kernel.h>
 
+LOG_MODULE_REGISTER(ll_schedule, LOG_LEVEL_INF);
+
 /* 1547fe68-de0c-11eb-8461-3158a1294853 */
-DECLARE_SOF_UUID("zll-schedule", zll_sched_uuid, 0x1547fe68, 0xde0c, 0x11eb,
+DECLARE_SOF_UUID("ll-schedule", zll_sched_uuid, 0x1547fe68, 0xde0c, 0x11eb,
```

Comment:
This is by mistake, thank you for spotting this. I was intended to declare a logging context `ll-schedule` both for src/schedule/zephyr_ll.c and src/schedule/ll_schedule.c. because ll_schedule.c is used with SOF+Zephyr on NXP SOCs.